week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,15 +1,28 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// Add comments to explain what this function does. You're meant to use Google!,  
 function getNumber() {
     return Math.random() * 10;
 }
+// A Math.random(), only displays "floting points"(witch is baseicly just numbers using to show result), 
+// numbers that are between 0 -1, 0 is shown but not 1. With approximantly unchangeble distribution.
+// You can then change those numbers to whatever you want. Then once you changed the scale.
+// It chouses a number between the scale at random, again not incloding the maxiem number but the minimem (in this case 0)
+// witch can also be included. However, in this case there is a " * 10" on line 3.
+// This means that what ever the algorithom choses the number, it will get multipiled by 10.
```

Comment:
This is a really good explanation of what this function is doing :) One more thing to think about - what are the limits of what this function may return? You've said that that `Math.random()` will return a value between 0 and 1 - what values may `getNumber()` return?